.github/workflows/test.yaml

```diff
@@ -90,6 +90,9 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v2
+      - name: lint
+        run: |
+          docker run --rm --interactive hadolint/hadolint < Dockerfile
```

Comment:
Neat, I had not heard of hadolint before today. Seems like a great project. 

Would it be possible to also add our release dockerfiles too?

https://github.com/pomerium/pomerium/tree/master/.github

Reply:
> Would it be possible to also add our release dockerfiles too?

sure, wanted to make sure that something like this was actually desired before adding the other `Dockerfile`s too